### PR TITLE
Minor fixes in getting started

### PR DESCRIPTION
**Pull request by rajarshimaitra**
Added some new links, and fixed some existing links and typos.

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
 # Bitcoin Dev Kit
 
-The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with a fully working *reference implementation* wallet called Magical Bitcoin.
+The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with a fully working *reference implementation* wallet called [Magical Bitcoin](https://github.com/MagicalBitcoin/android-app).
```
Comment (notmandatory) [inline docs/getting-started.md]: I think as a separate PR (see #84) we can do a list of all the reference projects we know of using bdk for mobile, desktop or servers. We can link here when that's ready, but for now:
```suggestion
The [Bitcoin Dev Kit (BDK)](https://github.com/bitcoindevkit) project (originally called Magical Bitcoin 🧙) aims to build a collection of tools and libraries that are designed to be a solid foundation for cross platform Bitcoin wallets, along with fully working *reference implementation* wallets for and various platforms.
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 All BDK components are designed to be lightweight and modular so that they can be adapted for virtually any use-case: from single-sig mobile wallets to multi-billion-dollar cold storage vaults.
 
 The main long-term goal is to concentrate the development efforts of multiple people and companies into one open source and very well reviewed project, instead of dispersing them over multiple closed/semi-closed or
 poorly designed projects.
 
-While some parts of the library are still considered "experimental" (check the docs for more info), the core `Wallet` architecture is now considered stable. We still can't commit to keeping this same exact API forever,
+While some parts of the library are still considered "experimental" (check the docs for more info), the core [`Wallet`](https://docs.rs/bdk/0.15.0/bdk/wallet/struct.Wallet.html) architecture is now considered stable. We still can't commit to keeping this same exact API forever,
```
Comment (notmandatory) [inline docs/getting-started.md]:
```suggestion
While some parts of the library are still considered "experimental" (check the docs for more info), the core [`Wallet`](https://docs.rs/bdk/latest/bdk/wallet/struct.Wallet.html) architecture is now considered stable. We still can't commit to keeping this same exact API forever,
```

**Comment by rajarshimaitra** [comment]
Thanks @notmandatory, updated with your suggestions.

**Comment by notmandatory** [comment]
Thanks!

**Pull request merged**